allennlp/nn/decoding/decoder_trainers/expected_risk_minimization.py

```diff
@@ -30,16 +30,22 @@ class ExpectedRiskMinimization(DecoderTrainer[Callable[[StateType], torch.Tensor
         al. do this in their work.
     max_decoding_steps : ``int``
         The maximum number of steps we should take during decoding.
+    max_num_decoded_sequences : ``int``, optional (default=1)
```

Comment:
Yeah, you put it as a constructor parameter here, where it should go, but then actually put the parameter in `decode`.

Reply:
I was going to originally make this a parameter to the constructor, but then realized that the loaded model would set this parameter from the archived config file. So I ended up taking the hacky path of modifying the instances in the script. But I like your idea of directly changing the decoder trainer's parameter in the script. Will do that, thanks!